buddyboss/buddyboss-platform
# Text formatting editor issues

### All Submissions:


Fixes #855
Fixes #964

### How to test the changes in this Pull Request:

1. Go to 'Forums', Activity, group message and Message screen
2. Check Editor

Follow this Video instruction: https://www.loom.com/share/a046d870296d4132b0760b19a2ff1fc4
### Other information:
